src/types.rs

```diff
@@ -0,0 +1,37 @@
+//! Re-exports of types for glossing over no_std/std distinctions
```

Comment:
I now see https://github.com/brson/error-chain/pull/64/files and it looks like it was using a similar hack...